src/EnergyPlus/ResultsFramework.cc

```diff
@@ -809,7 +821,7 @@ namespace ResultsFramework {
         }
     }
 
-    std::string &CSVWriter::convertToMonth(EnergyPlusData &state, std::string &datetime)
+    std::string &CSVWriter::convertToMonth(std::string &datetime)
```

Comment:
Remove state from the signature, it isn't needed without the fatal error call.

Reply:
:+1: